.devcontainer/Dockerfile

```diff
@@ -1,13 +1,35 @@
-FROM maven:3.6.3-jdk-11
+FROM maven:3.6.3-jdk-11-slim
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update
-RUN apt-get -y install --no-install-recommends apt-utils dialog dnsutils httpie bash-completion curl wget git unzip
-RUN apt-get -y install git procps lsb-release
+#Install .NET Core SDK Package Dependencies
+RUN curl https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb --output /tmp/packages-microsoft-prod.deb && \
+ dpkg -i /tmp/packages-microsoft-prod.deb
+
+#Install Common Tools
+RUN apt-get update && apt-get -y install --no-install-recommends \
+apt-transport-https \
+apt-utils \
+bash-completion \
+curl \
+dotnet-sdk-3.1 \
+dialog \
+dnsutils \
+git \
+httpie \
+lsb-release \
+procps \
+unzip
```

Comment:
Sorry to nitpick but if you could indent these lines out, it'll be easier to see that it's part of the same RUN command. Just speaking from experience working with the webv debug image PR.